openingd/dualopend.c

```diff
@@ -2871,8 +2878,8 @@ static void opener_start(struct state *state, u8 *msg)
 		master_badmsg(WIRE_DUALOPEND_OPENER_INIT, msg);
 
 	state->our_role = TX_INITIATOR;
-	tx_state->tx_locktime = tx_state->psbt->tx->locktime;
-
+	wally_psbt_get_locktime(tx_state->psbt, &locktime);
+	tx_state->tx_locktime = locktime;
```

Comment:
here and below you can just get it to `&tx_state->tx_locktime` directly.

Reply:
I think this might be correct, but depends on types, and emacs can't find the definition of get_locktime (probably a macro?)